src/app/content/studyGuides/hooks/closeStudyGuides.ts

```diff
@@ -0,0 +1,19 @@
+import { push } from '../../../navigation/actions';
+import { query } from '../../../navigation/selectors';
+import { removeParamFromQuery } from '../../../navigation/utils';
+import { ActionHookBody } from '../../../types';
+import { actionHook } from '../../../utils';
+import { modalQueryParameterName } from '../../constants';
+import { getContentParams } from '../../utils/urlUtils';
+import { closeStudyGuides } from '../actions';
+
+export const hookBody: ActionHookBody<typeof closeStudyGuides> = (services) => () => {
+  const state = services.getState();
+  const existingQuery = query(state);
+
+  services.dispatch(push(getContentParams(state), {
```

Comment:
i checked and apparently the content params are actually not in the navigation state. if you go with the option of just calling `pop` it won't matter, but if you do need to maintain access to the route params so that you can construct a new route without the search, you could consider adding them to the navigation state. i think i only didn't do that because i didn't feel like having them in two places, but this seems like a good use case of wanting them to be in a consistent place